pycognaize/tests/test_document/test_field/test_link_field.py

```diff
@@ -55,6 +65,7 @@ def test__construct_from_raw(self):
         self.assertEqual(self.link_field_1.name, 'Main Document')
         self.assertEqual(self.link_field_3.tags[0].raw_ocr_value, 'http://www.iaasa.ie/getmedia/b23890131cf6458b9b8fa98202dc9c3a/')
         self.assertNotEqual(self.link_field_3.tags[0].top, '45.5%')
+        self.assertEqual(self.link_field_5.name, 'table')
```

Comment:
is link_field_5 only used to test `.name` method, if so was not there another field already constructed that had a name attribute